files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml

```diff
@@ -10,8 +10,49 @@
 
   <operations>
 
-    <!-- Attach caption/transcript -->
+    <!-- Apply the default workflow configuration -->
+    <operation
+            id="defaults"
+            description="Applying default configuration values">
+      <configurations>
+        <configuration key="googleTranscript">true</configuration>
```

Comment:
We are not publishing google so this should be set to **false**

Reply:
Theses config settings are for the retract operation, not for publishing, therefore google and uploaded transcripts need to be set to true. 

This workflow attaches a Nibity transcript but before it does that, it removes Google captions and Uploaded captions.